app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/assets/FiatBalanceView.kt

```diff
@@ -18,13 +18,19 @@ import rdx.works.core.domain.assets.SupportedCurrency
 fun FiatBalanceView(
     modifier: Modifier = Modifier,
     fiatPrice: FiatPrice,
-    textStyle: TextStyle = RadixTheme.typography.body2HighImportance
+    textStyle: TextStyle = RadixTheme.typography.body2HighImportance,
+    decimalPrecision: Int? = null,
+    showAlways: Boolean = false,
```

Comment:
TBH, I am not sure what the `showAlways` does. 
It needs a more descriptive name.

Reply:
ah ok I checked this in your branch.
hmm it is a very special case, and I think to name it `isFee` makes it more distinct.

anyway it's not a hard requirement, as you wish 👌🏽

Reply:
currently `isFee` make sense, we should be aware that if we run into another scenario when we want fiat related value to show regardless of balance hiding feature, we will need to rename